pallets/staking/src/lib.rs

```diff
@@ -1150,6 +1150,9 @@ pub trait Trait:
 
     /// Yearly total reward amount that gets distributed when fixed rewards kicks in.
     type FixedYearlyReward: Get<BalanceOf<Self>>;
+
+    /// Maximum number of validators permitted.
+    type MaxValidatorAllowed: Get<u32>;
```

Comment:
Since this is only used in benchmarks and in runtime, it should be a constant in the benchmarks file, not here.

Reply:
We don't need to fix this as we are using this in the weight calculation. If you can see the weight function for `set_commission_cap`

Reply:
My point is that we are only using this in the benchmarking and weights. There's no reason why we can't just make it a constant there instead of making it a runtime constant.

Reply:
aah Ok